prompting/utils/uids.py

```diff
@@ -45,7 +42,7 @@ def check_uid_availability(
     return True
 
 
-def get_random_uids(self: BaseNeuron, k: int, exclude: list[int] = None) -> np.ndarray:
+def get_random_uids(k: int | None = 10**6, exclude: list[int] = None, own_uid: int | None = None) -> np.ndarray:
```

Comment:
We should make `k` mandatory, it's bounded to ~1000
```suggestion
def get_random_uids(k: int, exclude: list[int] = None, own_uid: int | None = None) -> np.ndarray:
```

Reply:
Why is that needed - there's only 1k miners in the network, right? So passing k=None simply means you sample all miners